internal/releaseapi/client.go

```diff
@@ -101,15 +106,54 @@ func (c *Client) DownloadRelease(r Release, os, arch string) (string, error) {
 		)
 	}
 
-	return c.downloadBuild(matchingBuild)
+	checkSums, err := c.getReleaseCheckSums(r)
+	if err != nil {
+		return "", errors.Wrap(err, "could not download checksum file")
+	}
+
+	for _, line := range strings.Split(checkSums, "\n") {
+		checksum := strings.Split(line, "  ")
+		if checksum[1] == matchingBuild.zipFileName() {
+			checkSha256Sum = checksum[0]
+			break
+		}
+	}
+
```

Comment:
Related to the other comment I left - should we fail if we didn't find the checksum?

Reply:
I'm worry about the string spit on double space "  " may broken in some release. This should be replaced with a better regex match. I will fix it and add the error handling logic if cannot find the checksum.